WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -1057,11 +1064,34 @@ public void onTokenInvalid() {
                     // we processed the voice reply, so we exit this function immediately
                     return;
                 } else {
-                    boolean shouldShowKeyboard =
-                            getIntent().getBooleanExtra(NotificationsListFragment.NOTE_INSTANT_REPLY_EXTRA, false);
-                    NotificationsListFragment
-                            .openNoteForReply(this, noteId, shouldShowKeyboard, null,
-                                    NotesAdapter.FILTERS.FILTER_ALL, true);
+                    if (mNotificationsViewModel == null) {
```

Comment:
I've added the initialisation here since the `initViewModel()` has not been called in this flow